.github/workflows/release.yml

```diff
@@ -185,6 +185,100 @@ jobs:
           inlineScript: |
             az storage blob upload-batch --account-name ${{ env.PREFIX }}public -d '$web' -s frontend/dist
 
+  deploy_release_test:
+    name: "Deploy Release: TEST"
+    needs:
+      - build_release
+      - build_frontend_release
+    if: github.ref == 'refs/heads/master'
+    environment: test
+    concurrency: test
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: prime-data-hub-test
+      ACR_REPO: pdhtestcontainerregistry.azurecr.io
+      PREFIX: pdhtest
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+
+      - name: Download Router Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: prime-router-build-${{ github.run_id }}
+          path: prime-router/build
+
+      - name: Download Frontend Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: static_website-${{ github.run_id }}
+          path: frontend/dist
+
+      - name: Build Docker Image
+        run: docker build . --file Dockerfile --tag ${{ env.ACR_REPO }}/${{ env.PREFIX }}:latest --tag ${{ env.ACR_REPO }}/${{ env.PREFIX }}:${{ github.sha }}
+
+      - name: Install OpenVPN
+        run: |
+          sudo apt-get update
+          sudo apt-get install openvpn
+          sudo apt install openvpn-systemd-resolved
+
+      - name: Connect VPN
+        uses: golfzaptw/action-connect-ovpn@master
```

Comment:
Is there a reason we are using `master` and not any one of https://github.com/golfzaptw/action-connect-ovpn/releases ?